protobufs/livekit_room.proto

```diff
@@ -61,10 +61,17 @@ service RoomService {
   // Update room metadata, will cause updates to be broadcasted to everyone in the room, Requires `roomAdmin`
   rpc UpdateRoomMetadata (UpdateRoomMetadataRequest) returns (Room);
 
-  // Forward a participant's track(s) to another room. Requires `roomAdmin` and `destinationRoom`. The forwarding will 
+  // Cloud-only
+  // a connected participant's track(s) to another room. Requires `roomAdmin` and `destinationRoom`. The forwarding will 
   // stop when the participant leaves the room or `RemoveParticipant` has been called in the destination room. 
   // A participant can be forwarded to multiple rooms. The destination room will be created if it does not exist.
   rpc ForwardParticipant(ForwardParticipantRequest) returns (ForwardParticipantResponse);
+
+  // Cloud-only
```

Comment:
Why it's cloud-only, though? It should be even easier to implement in OSS.

Reply:
rooms are not guaranteed to exist on the same server participant is connected to. this requires the mesh

Reply:
Makes sense to have it for single server setups at least.

Reply:
would not want to have features that only work under specific configuration & deployment. this would be a nightmare to support.